### PR TITLE
refactor: consistent route import for async and sync mode

### PR DESCRIPTION
**Pull request by shkreios**
# Current behavior
Async imports are handled inline
# New behavior
Async imports are handled the same way as sync imports

# Advantages
- Clean stringify code => no special cases
- Option in the future to allow preloaded

### CODE REVIEW

**Review by hannoeru — CHANGES_REQUESTED** [review]
Looks great, but can you revert all coding style changes that cause by prettier? I personally don't use prettier.

**Comment by shkreios** [comment]
> Looks great, but can you revert all coding style changes that cause by prettier? I personally don't use prettier.

I tried to configure prettier to look as similar as possible to your codestyle. Which styling tool are you using so I can easily revert the changes?

Thanks

File: src/stringify.ts [hunk 1]
```diff
@@ -39,40 +48,35 @@ export function stringifyRoutes(
...
+    if (content.startsWith('function')) return str.replace(replaceStr, content)
 
     if (content.startsWith('_NuFrRa_'))
       return str.replace(replaceStr, content.slice(8))
 
     return str
   }
 
-  const stringRoutes = JSON
-    .stringify(preparedRoutes, replaceFunction)
```
Comment (hannoeru) [inline src/stringify.ts:75]: revert this

**Comment by hannoeru** [comment]
Only ESLint, I don't want to have any coding style changes on the existing code in this PR.

**Comment by shkreios** [comment]
Ah sorry I thought you maybe had some tool, so contributors could easliy conform to the existing coding style.

**Comment by hannoeru** [comment]
Just add a vscode settings to disable prettier in this repo, thanks for revert those changes.

**Review by hannoeru — APPROVED** [review]

**Pull request merged**